src/node/api_clients/base_client.py

```diff
@@ -8,22 +8,17 @@
 
 
 class BaseClient(ABC):
-    # The retry decorator is applied to handle transient errors gracefully. This is particularly
-    # useful when running tests in parallel, where occasional network-related errors such as
-    # connection drops, timeouts, or temporary unavailability of a service can occur. Retrying
-    # ensures that such intermittent issues don't cause the tests to fail outright.
-    @retry(stop=stop_after_delay(0.5), wait=wait_fixed(0.1), reraise=True)
```

Comment:
Just out of curiosity, why is this no longer needed?

Reply:
I realized it makes more sense to handle retries case by case at higher level, instead of having all HTTP requests that fail retried by default.
There are more and more test cases where it is expected to have an error and I wanted to avoid retries for those cases.

So removing this retry gives the tests more control